BEIMA.Backend/MongoService/IMongoConnector.cs

```diff
@@ -106,6 +107,12 @@ public interface IMongoConnector
         public bool DeleteBuilding(ObjectId objectId);
 
         /// <summary>
+        /// Gets all devices that match the parameter filter's criteria
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <returns></returns>
+        
+        public List<BsonDocument> GetFilteredDevices(FilterDefinition<BsonDocument> filter);
```

Comment:
I don't think you merged this in properly, the comments and the spacing for everything is off.